.github/workflows/selenium.yml

```diff
@@ -70,4 +70,4 @@ jobs:
       - name: Run Selenium tests
         working-directory: ./tests/
         run: |
-          python -Wall runtests.py --verbosity 2 --noinput --selenium=chrome --headless --settings=test_postgres --parallel 2
```

Comment:
It will run in parallel by default, so we need to pass `--parallel 1` to avoid this.

Reply:
Also, thread errors are only on SQLite.